src/containers/AsideNavigation/InlineNavigation.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import {cn} from '@bem-react/classname';
+import type {RadioButtonOption} from '@gravity-ui/uikit';
+import {RadioButton} from '@gravity-ui/uikit';
+
+import {useNavigationMenuItems} from './useNavigationMenuItems';
+
+import './InlineNavigation.scss';
+
+const b = cn('inline-navigation');
+
+type MenuItem = ReturnType<typeof useNavigationMenuItems>[0];
```

Comment:
Why not `import type {MenuItem} from './useNavigationMenuItems';`?

Reply:
Cause typing is messed up again, MenuItem from the file is not what is returned by function